playwright-tests/takeDOMSnapshot.test.js

```diff
@@ -49,6 +49,28 @@ test('regular elements', async ({ page }) => {
       baseUrl: 'http://localhost:7700/regular-elements',
     },
   ]);
+
+  expect(snapshot.cssBlocks).toEqual([]);
+});
+
+test('style collection', async ({ page }) => {
+  await setupPage(page);
+
+  await page.goto('/style-collection');
+
+  const snapshot = await page.evaluate(() => {
+    return window.happoTakeDOMSnapshot({ doc: document, element: document.body });
+  });
+
+  expect(snapshot.cssBlocks.length).toBe(2);
+  expect(snapshot.cssBlocks[0].content).toMatch(/font-weight: 400;/);
+  expect(snapshot.cssBlocks[0].content).toMatch(/font: 400 1rem/);
```

Comment:
Should we do something here to make it more obvious that these are the variables? We could use names that don't match the styles exactly, and also maybe include the leading `--` in the assertions?

Reply:
Sounds good.